### ISSUE TITLE
ACL generator doesn't cover required cases

### ISSUE THREAD

**Issue by crandmck** [strongloop/generator-loopback#79]
I’m trying to create the ACL entry to allow authenticated users to write to the create property to recreate https://github.com/strongloop/loopback-getting-started-intermediate/blob/master/common/models/review.json#L44 as provided by @superkhau.

Here is what I entered:
```
$ slc loopback:acl
? Select the model to apply the ACL entry to: Review
? Select the ACL scope: A single method
? Enter the method name: create
? Select the role: Any authenticated user
? Select the permission to apply: Explicitly grant access
```

But in `review.json` this yields:
```
{
      "accessType": "EXECUTE",
      "principalType": "ROLE",
      "principalId": "$authenticated",
      "permission": "ALLOW",
      "property": "create"
    }
```
But it’s supposed to be 
```
{
      "accessType": "WRITE",
      "principalType": "ROLE",
      "principalId": "$authenticated",
      "permission": "ALLOW",
      "property": "create"
    },
```

The accessType should be `WRITE` not `EXECUTE`.

This affects the LoopBack Getting Started part 2 tutorial http://docs.strongloop.com/display/LB/Define+access+controls.
Please let me know when this is changed so I can update the docs.

**Comment by bajtos** [comment]
@ritch and/or @raymondfeng: you are most familiar with the ACL implementation, can you please comment?

**Comment by raymondfeng** [comment]
If the method name (i.e., `property`) is specified in the ACL entry, accessType should be `EXECUTE`. READ and WRITE can be used as wildcard. Please note now we allow strong-remoting to define accessType for each remote method to be READ or WRITE.

**Comment by crandmck** [comment]
So it appears that the generator is correct, and Simon fixed this in loopback-getting-started-intermediate and I made the corresponding fix in GS2.  

I reviewed [Controlling data access](http://docs.strongloop.com/pages/viewpage.action?pageId=3836269) but it's not clear if we need to change anything.  The responses that you give to `slc loopback:acl` are not affected, only what the generator puts in the JSON file.

Going to close this.
